pkg/types/health.go

```diff
@@ -61,16 +69,23 @@ func NewHealthConfig() *HealthConfig {
 func (c *HealthConfig) AddFlags(fs *flag.FlagSet) {
 
 	fs.BoolVar(&c.SnapshotLeaseRenewalEnabled, "enable-snapshot-lease-renewal", c.SnapshotLeaseRenewalEnabled, "Allows sidecar to renew the snapshot leases when snapshots are taken")
-	fs.BoolVar(&c.MemberLeaseRenewalEnabled, "enable-member-lease-renewal", c.MemberLeaseRenewalEnabled, "Allows sidecar to periodically renew the member leases when snapshots are taken")
+	fs.BoolVar(&c.MemberLeaseRenewalEnabled, "enable-member-lease-renewal", c.MemberLeaseRenewalEnabled, "Allows sidecar to periodically renew the member leases")
+	fs.BoolVar(&c.MemberGCEnabled, "enable-etcd-member-gc", c.MemberGCEnabled, "Allows leading sidecar to remove any superfluous etcd members from the cluster")
```

Comment:
```suggestion
	fs.BoolVar(&c.EtcdMemberGCEnabled, "enable-etcd-member-gc", c.MemberGCEnabled, "Allows leading sidecar to remove any superfluous etcd members from the cluster")
```
Here like this flag `--enable-etcd-member-gc` it is not ambiguous.